# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-display.html.ini
# Commit: 6141de941e71
# Full Hash: 6141de941e717cdb5485e1aa1ca00ffcd74f8a81
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 19:41:15
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 3cf752580f1b -r 6141de941e71 testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-display.html.ini
--- a/testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-display.html.ini	Fri Jan 29 19:20:24 2021 +0000
+++ b/testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-display.html.ini	Sat Jan 30 13:47:10 2021 +0000
@@ -1,10 +1,12 @@
 [legend-display.html]
-  [rendered legend with display: flow]
-    expected: FAIL
-
   [rendered legend with display: run-in]
     expected: FAIL
 
-  [rendered legend with display: inline]
+  [rendered legend with display: run-in; overflow:hidden]
     expected: FAIL
 
+  [rendered legend with display: run-in; columns:1]
+    expected: FAIL
+
+  [rendered legend with display: run-in; overflow:hidden;columns:1]
+    expected: FAIL